src/specs/api.v2.yaml

```diff
@@ -76,21 +107,110 @@ paths:
           content:
             application/json:
               schema:
-                $ref: ./models/HTTPSuccess.v1.yaml
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '401':
+          description: The request lacks authentication credentials.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+    patch:
```

Comment:
as I mentioned above, you are using "implicit" schema transformations for the patch/put operations, meaning that you just change the case of all the fields to update, and run the query. If we end up doing that, then the chemas here should be set to strict or those queries will crash if the UI sends unexpected or extra fields (which is fairly probable)

Reply:
I've seen you've done that in most of the cases with the `additionalProperties: false`. Do you know if it's possible to make it the default for all or something like that?

Reply:
I had looked at making it the default a long time ago, I didn't find any way to do that